### PR TITLE
Fixes deployable bug

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Fixes only one of each deployable item being deployable.

I did a whoops because I didn't realise the deployable element has a instance per item type, not per item.

Also fixes a couple other minor bugs/runtimes
## Why It's Good For The Game
Bug fix good
## Changelog
:cl:
fix: fixed a bug with deployable items not deploying and some other minor bugs
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
TM'ing so ungas can actually have more than one sentry.

File: code/datums/elements/deployable_item.dm [hunk 1]
```diff
@@ -28,13 +29,14 @@
...
 	UnregisterSignal(source, COMSIG_ITEM_EQUIPPED)
+	LAZYREMOVE(GLOB.deployable_items[source.type], source)
 
 ///Register click signals to be ready for deploying
 /datum/element/deployable_item/proc/register_for_deploy_signal(obj/item/item_equipped, mob/user, slot)
 	SIGNAL_HANDLER
 	if(slot != SLOT_L_HAND && slot != SLOT_R_HAND)
 		return
-	RegisterSignal(user, COMSIG_MOB_MOUSEDOWN, .proc/deploy)
+	RegisterSignal(user, COMSIG_MOB_MOUSEDOWN, .proc/deploy, TRUE)
```
Comment (TiviPlus) [inline code/datums/elements/deployable_item.dm:39]: You need to justify this

File: code/datums/elements/deployable_item.dm [hunk 2]
```diff
@@ -19,7 +19,8 @@
 	deploy_time = _deploy_time
 	undeploy_time = _undeploy_time
 
-	attached_item = target
+	var/obj/item/attached_item = target
+	LAZYADD(GLOB.deployable_items[attached_item.type], attached_item)
```
Comment (TiviPlus) [inline code/datums/elements/deployable_item.dm:23]: ugly but whatever

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/datums/elements/deployable_item.dm (hunk 1 above)
Comment (Lumipharon) [reply code/datums/elements/deployable_item.dm:39]: Holding two deployables

Comment (TiviPlus) [reply code/datums/elements/deployable_item.dm:39]: No with a comment

Comment (TiviPlus) [reply code/datums/elements/deployable_item.dm:39]: Also should probably be a component but w/e

Comment (Lumipharon) [reply code/datums/elements/deployable_item.dm:39]: Spooky said keep it as an element. Added the comment though.

**Pull request merged**